common/prettify.pm

```diff
@@ -620,24 +623,36 @@ sub Footer ()
 {
     my $self = shift;
     my $out = $self->{FH};
+    my $host = hostname;
```

Comment:
The host running this script is not necessarily the host that ran the tests.

Reply:
We did have setups where we used multiple hosts for running the TAO unit tests, client on one embedded board, server on another

Reply:
replaced with the hostname in output.log_Config.html